rs/registry/canister/src/mutations/node_management/common.rs

```diff
@@ -55,7 +55,7 @@ pub fn get_subnet_list_record(registry: &Registry) -> SubnetListRecord {
     SubnetListRecord::decode(subnet_list_record_vec.as_slice()).unwrap()
 }
 
-pub fn get_node_operator_id_for_node(
+pub fn get_node_operator_id_for_node_id(
```

Comment:
I think get_node_operator_id_for_node makes sense, given that's what it's looking up.  I think the below could be get_node_provider_id_for_node_operator, and the parameter name (and type) tells you that you have to provide the ID in order for the function to work.

But logically, you're looking up an ID that relates to an object, and the object is identified by another ID.

Reply:
Would the same logic hold for the `node_operator_id`? I tried to improve consistency by naming the function according to what it takes - it takes a node id, and not a node record/object. Yes, a node is determined by the node id, but the same holds for other functions and objects, such as `node_operator_id`.
So for me it would be more logical to either drop `id` for all parameters, or have it for all.

Reply:
I'm not saying drop it for the parameter names, just the function name.

Reply:
Understood, done.